src/Popover/index.jsx

```diff
@@ -20,15 +20,15 @@ const WrapperPopover = React.forwardRef(({
 }, ref) => (
   <Popover
     {...props}
-    className={classNames(`popover-${variant}`, props.className)}
+    className={classNames({ [`popover-${variant}`]: !!variant }, props.className)}
```

Comment:
If `variant` isn't provided (i.e., it's an optional prop), don't include a `popover-*` class name in the DOM.